client/src/components/Menu/Menu.scss

```diff
@@ -0,0 +1,638 @@
+/**
+ * Styles for the left hand side menu and header for the admin panels.
+ *
+ * Take into consideration CSS selector performance.
+ *
+ * @package framework
+ * @subpackage admin
+ */
+
+.cms-menu {
+  z-index: $zindex-menu;
+  background: $color-theme-bg;
+  // Add `!important` to ignore width set in `.cms-panel` entwine`
+  width: $cms-menu-width !important;
```

Comment:
was it not possible to remove the width set in entwine?
Just getting a little bit of context, as this gives more warnings in our linting